config/install/filter.format.az_standard.yml

```diff
@@ -0,0 +1,62 @@
+langcode: en
```

Comment:
The AZ Standard text format is already included in az_paragraphs/config/optional.  You'll want to update the existing exported config rather than add a new one here.  Ultimately we probably need to find a more intuitive home for our text formats (maybe az_core) but that can be a separate improvement from this PR.

Reply:
Thanks! Missed this